docs/views/templates/check-your-answers.html

```diff
@@ -6,15 +6,22 @@
   Check your answers template - NHS.UK prototype kit
 {% endblock %}
 
+{% block beforeContent %}
+  {{ backLink({
+    "href": "/templates/start-page",
+    "text": "Back"
+  }) }}
+{% endblock %}
+
 {% block content %}
   <div class="nhsuk-grid-row">
     <div class="nhsuk-grid-column-two-thirds">
 
-      <h1 class="nhsuk-u-margin-top-4">
+      <h1 class="nhsuk-heading-l">
```

Comment:
I wonder if we should redefine `h1`, `h2`, and `h2` in scss so that these don't need to be set on every heading.

The pro would be that prototypes get the right defaults for transactional services - and reduce the risk of designs accidentally using the wrong one. The con would be that they _might_ be surprised a not needing a class when working on these yet on production the font size is wrong.

Reply:
@edwardhorsford one for the backlog? I'm not so sure about it - kinda think we should be as similar to production as possible (often catches people out with the govuk prototype kit)

Reply:
Ok, let's leave for now - but consider how we can improve this in the future.